Bridge/cctp/src/utils/tracking.ts

```diff
@@ -0,0 +1,154 @@
+import { Instance } from "@blockflow-labs/utils";
+import BigNumber from "bignumber.js";
+
+const today = new Date();
+const date = today.toISOString().split("T")[0];
+const week = Math.floor(Date.now() / 604800000);
+const month = new Date().getMonth();
+const year = new Date().getFullYear();
+
```

Comment:
We can't use this to calculate, we need to use block.timestamp for all calculations. 

For example, if an event happened last month CCTP, we are processing and counting it in today timestamp.